.devops/code-review-pipelines.yml

```
@@ -55,3 +55,19 @@ stages:
               SUBSCRIPTION: PROD-IO-SERVICE-CONN
               ENVIRONMENT: prod
               WORKINGDIR: 'src/core'
+      - job: validate_migration
+        steps:
+          - checkout: self
+          - task: AzureCLI@2
+            displayName: validate migration data on subscription-migrations
+            inputs:
+              azureSubscription: PROD-IO-SERVICE-CONN
+              scriptType: 'bash'
+              scriptLocation: 'inlineScript'
+              failOnStandardError: false
+              inlineScript: |
+                set -e
+                cd src/psql/selfcare/subscription-migrations
+                echo "Validating migrations"
+                ./flyway.sh validate PROD-IO db && echo "Validation successful" || { echo "Validation failed"; exit 1; }
```

```suggestion
                ./flyway.sh info PROD-IO db && echo "Validation successful" || { echo "Validation failed"; exit 1; }
```

> from flyaway docs: https://flywaydb.org/documentation/command/validate
> The validate mechanism checks if the migration locally still has the same checksum as the migration already executed in the database.
> 

So when we add sql migrations is expected that validate fails. Probably we need to add `-ignorePendingMigrations=true` or use `info` command

---

Dunno, let's try - good insight indeed!